app/views/templates/ButtonGroup.scala.html

```diff
@@ -0,0 +1,27 @@
+@*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@this(govukButton: GovukButton)
+@(continueMessage: String, period: Period)(implicit messages: Messages)
+<div class="govuk-button-group">
+    @govukButton(
+    ButtonViewModel(messages(continueMessage))
+    )
+
+    <a role="button" href="@routes.SavedProgressController.onPageLoad(period)" class="govuk-button govuk-button--secondary" data-module="govuk-button">
+        @messages("saveProgress.button")
+    </a>
+</div>
```

Comment:
I think this should live in components, not templates

Reply:
moved now